sdk/identity/azure-identity/tests/test_win_vscode_credential_async.py

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import sys
+import pytest
+from azure.core.credentials import AccessToken
+from helpers_async import wrap_in_future
+if sys.platform.startswith('win'):
+    from azure.identity.aio._credentials.win_vscode_credential import WinVSCodeCredential
+    from azure.identity._credentials.win_vscode_credential import _cred_write
+try:
+    from unittest.mock import Mock
+except ImportError:  # python < 3.3
+    from mock import Mock  # type: ignore
```

Comment:
We're definitely on Python 3.3+:
```suggestion
from unittest.mock import Mock
```